the-collab-lab/tcl-25-smart-shopping-list
# [WIP] Issue #1 | Connect app to Firebase

## Description


This PR  adds a component with a button that increments a number in the database. It ensures that changes made in the Firestore database get displayed in the app as well as updates made in the app show up in the Firestore database

## Acceptance Criteria

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|  ✓  | :link: Update dependencies |
|    | :scroll: Docs              |

## Testing Steps / QA Criteria
* From your terminal, pull down this branch with `git pull origin ku-io-connect-to-firebase` and check that branch out with `git checkout ku-io-connect-to-firebase` 
* Then `npm` to install the newly added dependencies locally and `npm start` to launch the app.
* Click on the Increment button. Make sure it increments the counter value and make sure that value appears in the `counters` collection of the database [here](https://console.firebase.google.com/u/0/project/tcl-25-shopping-list/firestore/data~2F).
